app/assets/stylesheets/replies.scss

```diff
@@ -19,33 +19,34 @@
   color: $font_color_icon_selector;
 
   select {
-    width: 100px;
     margin-bottom: 10px;
+    width: 100px;
   }
 }
 
 /* Character selector */
 .selector-popup-button {
-  width: 14px;
-  height: 14px;
   position: relative;
   top: 1px;
   cursor: pointer;
+  width: 14px;
+  height: 14px;
 }
 
 .selector-popup {
+  display: none;
   position: absolute;
   top: 0;
   left: 150px;
-  padding: 5px 20px 10px 20px;
   z-index: 2;
+  background-color: $bg_color_selectionpopup;
```

Comment:
Where did this rule come from?

Reply:
If i had to guess I'd say I probably went through and found what it was rendering at on the default b/c some themes set it. I don't... know why it's in this commit tho

Reply:
apparently it came from merging in another selector